source/agora/common/crypto/ECC.d

```diff
@@ -20,6 +20,7 @@ module agora.common.crypto.ECC;
 import agora.common.Hash;
```

Comment:
The commit message needs a description and a note of which issue it fixes. For example:

```
Emit hashed version of a secret key by default

To avoid leaking private keys into logs, we emit a shortened hashed version of the key. The behavior can toggled off with the `PrintMode` argument to `toString`.

Fixes #719
```